core/templates/dev/head/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -636,6 +678,26 @@ angular.module('oppia').directive('stateResponses', [
             });
 
             $scope.$on('updateAnswerChoices', function(evt, newAnswerChoices) {
+              var interactionId = $scope.getCurrentInteractionId();
+              if (interactionId === 'MultipleChoiceInput') {
+                var oldAnswerChoices = ResponsesService.getAnswerChoices();
+                if (newAnswerChoices !== null && oldAnswerChoices !== null) {
+                  var deletedChoicesIndex = getDeletedChoicesIndex(
```

Comment:
The naming of this is confusing. "Index" is singular but you seem to be treating it as a plural. "Indexes", perhaps?

Reply:
Not relevant anymore